pnpm-lock.yaml

```diff
@@ -3632,8 +3632,8 @@ packages:
   axios@0.28.1:
```

Comment:
>axios@0.28.1 [](http://example.com/codeflow?start=2&length=12)

For others who might look at this later, it is okay that we still depend on axios 0.28.1 as the vulnerable versions only start with 1.x. So 0.28.1 is okay (at least as far as this particular vulnerability is concerned.)